src/Bookings.js

```diff
@@ -1,18 +1,17 @@
 import React from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+import FakeBookings from "./data/fakeBookings.json";
```

Comment:
Since this isn't a component, it is good convention to use camelcase naming for this. So `fakeBookings` would be more suitable. 

Try to reserve capital casing for React components.

This is a minor very minor comment 👍

Reply:
Yes it's a good point